bpf/bpf_lxc.c

```diff
@@ -72,6 +72,7 @@
 #ifdef ENABLE_PER_PACKET_LB
 
 #ifdef ENABLE_IPV4
+
```

Comment:
Please revert these whitespace changes, there are more in the file.

Reply:
Done